docs/troubleshooting-guide/application-database.md

```diff
@@ -1,75 +1,123 @@
-# Trouble with the Metabase Application Database
+---
+redirect_from:
+  - ./loading-from-h2.html
+---
+# The application database isn't working
```

Comment:
I would change this in some way, how does anyone know that the application database isn't working?